frontend/src/__tests__/cypress/cypress/e2e/notebookImageSettings/NotebookImageSettings.cy.ts

```diff
@@ -182,13 +189,46 @@ describe('Notebook images', () => {
     importNotebookImageModal.findAddPackagesButton().click();
     importNotebookImageModal.findCancelButton().should('be.disabled');
     importNotebookImageModal.findSubmitButton().should('be.disabled');
+
+    //succesfully import notebook image
+    importNotebookImageModal.findPackageseNameInput().type('packages-3');
+    importNotebookImageModal.findPackagesVersionInput().type('version');
+    importNotebookImageModal.findSaveResourceButton().click();
+    importNotebookImageModal.findSubmitButton().should('be.enabled');
+
+    importNotebookImageModal.findSubmitButton().click();
+
+    cy.wait('@importNotebookImage').then((interception) => {
+      expect(interception.request.body).to.eql({
+        /* eslint-disable-next-line camelcase */
```

Comment:
I tried to disable camel case for this file, but for some reason its not working. So disabled individual line here.

Reply:
You can also do this at the top of the file: `/* eslint-disable camelcase */`
It's often better to just do it per line though.

I wonder if there's a better typescript rule that we can configure so that we disable globally for specific type properties. Maybe we can configure [this rule](https://github.com/typescript-eslint/typescript-eslint/blob/main/packages/eslint-plugin/docs/rules/naming-convention.md) to do that. But for now what you've fine is fine.